src/NuGet.Clients/NuGet.PackageManagement.UI/Converters/PackageManagerControlLeftSideMaxWidthConverter.cs

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Globalization;
+using System.Windows;
+using System.Windows.Data;
+
+namespace NuGet.PackageManagement.UI
+{
+    public class PackageManagerControlLeftSideMaxWidthConverter : IMultiValueConverter
```

Comment:
not clear from name what it converts it into. you also don't need to include PackageManagerControl in the name

Reply:
It is taking the first value and substracting the second value to it. This was the way I was able to get the `MaxWidth` of the left side of the UI by substracting the `rightSide + gridSplittler` from the current `ActualWidth` of the UI. What name would you suggest for it? Is something like `ActualWidthToLeftSideMaxWidthConverter` expressive enough?

Reply:
for the name, I would say just use ActualWidthToMaxWidthConverter . but add a description in the class that explains it like in the above comment.